February 11th Cloud Release Notes (#375)
astronomer/docs: cloud/release-notes.md

**Latest CLI Version**: 1.0.0 ([Release notes](cli-release-notes.md))

## February 11, 2022

### Monitor DAG Runs Across All Deployments in a Workspace

> You can view key metrics about recent DAG runs through the new **DAGs** page in the Astronomer Cloud UI. Use this page to view DAG runs, including successes and failures, across all of your Deployments at a glance, or drill down to a specific DAG and see metrics about its recent runs.

Reviewer: ```suggestion
You can view key metrics about recent DAG runs through the new **DAGs** page in the Astronomer Cloud UI. Use this page to view DAG runs at a glance, including successes and failures, across all Deployments in a given Workspace. You can also drill down to a specific DAG and see metrics about its recent runs.
```

![DAGs page](/img/docs/dags-page.png)

For more information about the **DAGs** page, see [Deployment Metrics](deployment-metrics#dag-runs).

### Additional Improvements

> - Added more ways to sort pages with card views, such as the **Deployments** page

Author: ```suggestion
- Added more ways to sort pages that utilize card views, such as the **Deployments** page
```

> - Added more ways to sort pages that utilize card views, such as the **Deployments** page
> - Added user avatars next to usernames in several places across the UI
> - A banner now appears in the UI if a Deployment is running an unsupported version of Astronomer Runtime
> - Equivalent CPU/GB usage now appears next to all AU sliders when configuring a Deployment's resource settings

Reviewer: ```suggestion
- All resource settings in the Deployment view of the Astronomer UI now show exact CPU and Memory usage to the right of every slider, previously shown only in Astronomer Units (AUs). This makes it easy to know exactly how many resources you allocate to each component.
- A banner now appears in the Astronomer UI if a Deployment is running a version of Astronomer Runtime that is no longer maintained. To make the most of features and bug fixes, we encourage users to upgrade to recent versions as much as possible.
- Added more ways to sort pages that utilize card views, such as the **Deployments** page
- Added user account avatars next to usernames in several places across the Astronomer Cloud UI
```

### Bug Fixes

> - Removed a non-functional **Environment** field from a Deployment's information page

Reviewer: ```suggestion
- Removed the **Environment** field from the Deployment view of the Astronomer UI. This field is not currently functional and will be re-added as soon as it is.
```
